src/lib/containers/home.tsx

```diff
@@ -85,7 +85,10 @@ export class Home extends React.Component<Props, State> {
   }
 
   componentDidUpdate() {
-    this.listView.scrollTo({ y: this.currentScrollOffset + 1, animated: false })
+    if (this.currentScrollOffset < 0) {
```

Comment:
Does that mean that the list will scroll back to 0 immediately after releasing the drag gesture, as opposed to after the network fetch completed?